### PR TITLE
Advance configuration in admin portal and life cycle support for API Products

### PR DESCRIPTION
**Pull request by Tharanidk**
This PR consist of two test cases as follows,

01.Advanced configurations - Admin Portal
02.Life cycle support for API Products

### CODE REVIEW

**Review by chanaka3d — APPROVED** [review]

File: tests/cypress/integration/admin/10-advanced-configurations.specs.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            body: adavanceConfTrueJson
+        })
+        cy.get('[data-testid="Scope Assignments-child-link"]').click();
+        cy.get('[data-testid="Advanced-child-link"]').click();
+        cy.wait(2000);
+        cy.get('[data-testid="monaco-editor-save"]').click();
+
+    });
+
+ })
```
Comment (HiranyaKavishani) [inline tests/cypress/integration/admin/10-advanced-configurations.specs.js:54]: Add a new line after EOF

File: tests/cypress/integration/publisher/013-api-product/04-lifecycle-support-for-api-products.specs.js [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+        });
+    });
+
+    after(function () {
+
+        //Delete Users
+        cy.visit(`${Utils.getAppOrigin()}/carbon/user/user-mgt.jsp`);
+        cy.deleteUser(publisher);
+    })
+})
```
Comment (HiranyaKavishani) [inline tests/cypress/integration/publisher/013-api-product/04-lifecycle-support-for-api-products.specs.js:133]: Add a new line after EOF

File: tests/cypress/integration/admin/10-advanced-configurations.specs.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import adavanceConfFalseJson from "../../fixtures/api_artifacts/advanceConfigFlase.json"
```
Comment (hisanhunais) [inline tests/cypress/integration/admin/10-advanced-configurations.specs.js:19]: Shall we fix the spelling mistake here? ad**a**vanceConfFalseJson
 Better to check the usages of this and fix those as well. The file name also will have to be renamed. advanceConfig**Flase**.json

File: tests/cypress/integration/admin/10-advanced-configurations.specs.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+import adavanceConfTrueJson from "../../fixtures/api_artifacts/advanceConfigTrue.json"
```
Comment (hisanhunais) [inline tests/cypress/integration/admin/10-advanced-configurations.specs.js:20]: Shall we fix the spelling mistake here? ad**a**vanceConfTrueJson. Better to check the usages of this and fix those as well.

File: tests/cypress/integration/admin/10-advanced-configurations.specs.js (hunk 1 above)
Comment (Tharanidk) [reply tests/cypress/integration/admin/10-advanced-configurations.specs.js:54]: Resolved with commit e159e6db870377cf9dc4726b102cf6292273660e

File: tests/cypress/integration/publisher/013-api-product/04-lifecycle-support-for-api-products.specs.js (hunk 2 above)
Comment (Tharanidk) [reply tests/cypress/integration/publisher/013-api-product/04-lifecycle-support-for-api-products.specs.js:133]: Resolved with commit e159e6db870377cf9dc4726b102cf6292273660e

File: tests/cypress/integration/admin/10-advanced-configurations.specs.js (hunk 3 above)
Comment (Tharanidk) [reply tests/cypress/integration/admin/10-advanced-configurations.specs.js:19]: Resolved with commit e159e6db870377cf9dc4726b102cf6292273660e

File: tests/cypress/integration/admin/10-advanced-configurations.specs.js (hunk 4 above)
Comment (Tharanidk) [reply tests/cypress/integration/admin/10-advanced-configurations.specs.js:20]: Resolved with commit e159e6db870377cf9dc4726b102cf6292273660e

**Pull request merged**